docs/Home.md

```diff
@@ -0,0 +1,88 @@
+# Google Kubernetes Engine Plugin Wiki
+
+The [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/) (GKE) Plugin allows you to publish deployments built within Jenkins to your Kubernetes clusters running within GKE.
+
+_Plugin Information_
+
+View Google Kubernetes Engine [on the plugin site](https://plugins.jenkins.io/google-kubernetes-engine) for more information.
+
+## Usage
+
+### IAM Credentials
+
+1. Create a service account using the Google Cloud SDK.
+    ```bash
+    gcloud iam service-accounts keys create ~/jenkins-gke-key.json --iam-account jenkins-gke@${PROJECT}.iam.gserviceaccount.com
+    ```
+1. Add the KubernetesEngineAdmin, serviceAccountUser, and computeNetworkViewer roles to the service account:
+    ```bash
+    export PROJECT=$(gcloud info --format='value(config.project)')
+    export SA_EMAIL=$(gcloud iam service-accounts list --filter="name:jenkins-gke" --format='value(email)')
+    gcloud projects add-iam-policy-binding --member serviceAccount:$SA_EMAIL --role roles/iam.serviceAccountUser $PROJECT
+    gcloud projects add-iam-policy-binding --member serviceAccount:$SA_EMAIL --role roles/container.clusterAdmin $PROJECT
+    gcloud projects add-iam-policy-binding --member serviceAccount:$SA_EMAIL --role roles/container.admin $PROJECT
+    gcloud projects add-iam-policy-binding --member serviceAccount:$SA_EMAIL --role roles/compute.networkViewer $PROJECT
+    ```
+1. Download a JSON Service Account key for your newly created service account. Take note of where the file was created, you will upload it to Jenkins in a subsequent step.
+    ```bash
+    gcloud iam service-accounts create jenkins-gke
+    ```
+1. In Jenkins, click the Credentials button on the left side of the screen. Then click System.
+1. Click Global credentials then **Add credentials** on the left.
+1. In the Kind dropdown, select **Google Service Account from private key**.
+1. Enter your project name then select your JSON key that was created in the preceding steps.
+1. Click OK.
+
+### Enable Cloud Resource Manager API
+
+1. Go to the [Cloud Resource Manager API](https://console.developers.google.com/apis/api/cloudresourcemanager.googleapis.com/overview) overview in the GCP console.
```

Comment:
It might make sense to have this step use gcloud as well to keep it consistent with the rest of the doc. Also they'll probably need to enable the compute and GKE APIs: gcloud services enable compute.googleapis.com container.googleapis.com servicemanagement.googleapis.com

Reply:
Done